### ISSUE TITLE
WARNING:root:Trying to call reset() of MultiStepLIFNode

### ISSUE THREAD

**Issue by AragornHorse** [ZK-Zhou/spikformer#28]
WARNING:root:Trying to call reset() of MultiStepLIFNode( v_threshold=0.5, v_reset=0.0, detach_reset=True, tau=2.0, backend=cupy (surrogate_function): Sigmoid(alpha=4.0, spiking=True) ), which is not spikingjelly.activation_based.base.MemoryModule

Have you met this warning before? How to solve it.

**Comment by Extra-fan** [comment]
Hello, I met the same problem and I figured it out. I would like to share it with you.
**SOLUTIONS**
change your import path
```python
# the import info in your code
from spikingjelly.activation_based import functional

# use the code below
from spikingjelly.clock_driven import functional
```
**ANALYSIS**
First, **check your SpikingJelly verison**, if the version `<=0.0.0.0.12`, it's the **old version**, or it's the **new version**
As the official document mentioned, new version renamed some sub-packages.
`clock_driven ====>>>> activation_based`
In a new version, you can find function `reset_net()` in `spikingjelly.activation_based.functional`, this is source code without explanation
```python
def reset_net(net: nn.Module):
    for m in net.modules():
        if hasattr(m, 'reset'):
            if not isinstance(m, base.MemoryModule):
                logging.warning(f'Trying to call `reset()` of {m}, which is not spikingjelly.activation_based.base'

                                f'.MemoryModule')
            m.reset()
```
And in the old one, the code is
```python
def reset_net(net: nn.Module):
    for m in net.modules():
        if hasattr(m, 'reset'):
            m.reset()
```
It is evidence that new version check the class type of the input ,while the old one doesn't.
`class MultistepLIFNode()` is defined in `clock_driven` which is used in old version SpikingJelly. So you should use the old version `reset_net()` which is defined in `spikingjelly.clock_driven.functional`

For More Information Please Visit:[从老版本迁移 — spikingjelly alpha 文档](https://spikingjelly.readthedocs.io/zh-cn/latest/activation_based/migrate_from_legacy.html)
